### PR TITLE
Store contact in fineract correctly

### PR DESCRIPTION
**Pull request by sagarrakshe**

### CODE REVIEW

File: user_client.go [hunk 1]
```diff
@@ -12,6 +13,8 @@ type UserInfoResponse struct {
 	LastName    string `json:"lastname"`
 	DisplayName string `json:"displayname"`
 	MobileNo    string `json:"mobileNo"`
```
Comment (jupiter-ts) [inline user_client.go]: I think we can remove `mobileNo` so thing will not get confused.

File: user_client.go [hunk 2]
```diff
@@ -22,5 +25,16 @@ func (client *Client) GetClientInfo(clientId string) (*UserInfoResponse, error)
...
+	//NOTE: For backward compatibility error is not being raised here, since many numbers are stored without country-code
+	contact := strings.Split(response.MobileNo, "_")
+	if len(contact) < 2 {
+		response.PhoneNumber = response.MobileNo
+		return response, nil
+	} else {
+		response.CountryCode = contact[0]
+		response.PhoneNumber = contact[1]
+		return response, nil
+	}
```
Comment (sapanakothari) [inline user_client.go]: rename this method to GetClient

File: client.go [hunk 3]
```diff
@@ -61,10 +62,18 @@ func (client *Client) CreateClient(clientInfo *ClientInfo, merchantUserID string
 			},
 		},
 	}
+
+	request.PhoneNumber = ""
+	request.CountryCode = ""
```
Comment (jupiter-ts) [inline client.go]: I think you can use ```CountryCode    string    `json:"-"```` to omit the key from json serialization

Comment (sagarrakshe) [reply client.go]: 👍

**Pull request merged**